food-truck-frontend/src/components/RouteList.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import api from "../util/api";
+import RouteListRow from "./RouteListRows";
+import {Button} from "@material-ui/core";
+import CreateRouteRow from "./CreateRouteRow";
+
+const TruckObject = {
+        id: "",
+        userId: "",
+        name: "",
+        schedule: "",
+}
+
+const RouteObject = [{
+    routeId: 0,
+    routeName: "",
+    active: ""
+}]
+
+
+
+
+type RouteState = {
+    truck: {
+        "id": string,
+        userId: string,
+        name: string,
+        schedule: string,
+    },
+    routeData: {
+        routeId: number
+        routeName: string
+        active: string
+    }[],
+    startCreate: boolean
+}
+
+
+type RouteProps = {
+    truckId: string
+}
+
+class RouteList extends React.Component<RouteProps, RouteState>{
+    constructor(props: RouteProps) {
+        super(props);
+        this.state = {
+            truck: TruckObject,
+            routeData: RouteObject,
+            startCreate: false,
+        }
+
+    }
+
+    componentDidMount(){
+        api.get(`/truck/${this.props.truckId}`).then(res => {
+            this.setState({truck : res.data});
+        }).catch(err => {
+            console.log(err);
+            this.setState(err.toString())
+        });
+
+        this.fetchRoutes();
+    }
+
+    deleteRow(routeId: string){
+        api.delete(`/truck/routes-delete/${routeId}`).then(() =>{
+            this.fetchRoutes();
+        }).catch(err =>{
+            console.log(err.toString())
```

Comment:
Not sure that we want to be using `console.log(..)` on master, but if we think we should then OK

Reply:
I know I always automatically throw in error logging like this. I'd probably be better to switch it to something more professional, though. I'll throw an issue for it on Backlog.